packages/core/content-type-builder/admin/src/pages/App/index.tsx

```diff
@@ -34,9 +33,10 @@ const App = () => {
     }
   }, []);
 
+  // FIXME Error here
```

Comment:
Can we remove this?

Reply:
.... no, I think we probs need to to remember that this page, is less than desirable and needs some love.